product/timelines/miq_reports/tl_events_hourly.yaml

```diff
@@ -67,6 +68,7 @@ col_order:
 - container_namespace
 - container_name
 - container_replicator_name
+- middleware_name
```

Comment:
No, I'm fine with that approach. I was just confused because you used
middleware_server_name in the event_hash below.

Am Dienstag, 26. April 2016 schrieb Jay Shaughnessy :

> In product/timelines/miq_reports/tl_events_hourly.yaml
> <https://github.com/ManageIQ/manageiq/pull/7969#discussion_r61151177>:
>
> > @@ -67,6 +68,7 @@ col_order:
> >  - container_namespace
> >  - container_name
> >  - container_replicator_name
> > +- middleware_name
>
> This is a slight departure from the way it's done in other places. For
> example, above you can see several container_* entries. And then analogous
> entries in timeline.rb (and also ems_event.rb). I avoided hard-coding in
> those files, so that as we add more entities to the mw provider we don't
> need to keep remembering to update the .rb files. Instead I just have a
> single entry here to handle the single entity object that may be provided
> for the event. In the event pop-up it will have: "Middleware Resource: ".
> If you guys totally hate this approach I can change it to be hard-coded and
> require update as we go, but hopefully it's OK.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/7969/files/8fc3a50d28b5588865e5cb34886e2ddfcd5a376e#r61151177>
>